app/mixins/move_actions.js

```diff
@@ -23,31 +21,27 @@ const ALLOW_PARTIAL_QTY = config.APP.ALLOW_PARTIAL_OPERATIONS;
  */
 export default Ember.Mixin.create(AsyncMixin, {
   locationService: Ember.inject.service(),
+  settings: Ember.inject.service(),
+
+  editableQty: Ember.computed.alias("settings.allowPartialOperations"),
 
-  editableQty: ALLOW_PARTIAL_QTY,
-
-  moveQty: Ember.computed(
-    "_moveQty",
-    "moveFrom",
-    "moveTarget",
-    "moveTarget.packagesLocations.@each.quantity",
-    {
-      get(k) {
-        return this.get("_moveQty");
-      },
-      set(k, value) {
-        const total = this.quantityAtSource();
-        const qty = Number(value);
-
-        if (!ALLOW_PARTIAL_QTY && qty > 0 && qty !== total) {
-          throw new Error("Partial quantity is not permitted");
-        }
-
-        this.set("_moveQty", qty);
-        return qty;
+  moveQty: Ember.computed("_moveQty", {
+    get(k) {
+      return this.get("_moveQty");
+    },
+    set(k, value) {
+      const allowPartial = this.get("settings.allowPartialOperations");
```

Comment:
can we use `editableQty` here instead ? as it is already set above.

Reply:
We could but I'd rather not, I left it that way on purpose for readability.

Reading `if notEditable then checkIsPartial` doesn't make sense
whereas reading `if !allowPartial then checkIsPartial`  does.

Even if both variables come from the same source, they hold different meanings in their names :)